config/application.rb

```diff
@@ -69,5 +69,9 @@ class Application < Rails::Application
     config.after_initialize do |app|
       app.routes.append { get '*path', to: 'errors#not_found' }
     end
+
+    config.action_dispatch.default_headers = {
+      'Feature-Policy' => "accelerometer 'none'; camera 'none'; geolocation 'none'; gyroscope 'none'; magnetometer 'none'; microphone 'none'; payment 'none'; usb 'none'",
```

Comment:
Do we want to change to `Permissions-Policy`?

https://developer.mozilla.org/en-US/docs/Web/HTTP/Feature_Policy

Reply:
It might be worth giving the list of possible permissions a once-over in case they've added anything since the pen test: https://developer.mozilla.org/en-US/docs/Web/HTTP/Feature_Policy#granular_control_over_certain_features.

Reply:
Good questions. It looks like `Feature-Policy` is better supported than `Permissions-Policy`

https://caniuse.com/?search=Feature-policy - All the modern browsers support it
https://caniuse.com/?search=Permissions-policy - Just recent Chrome and Edge

Looks like there are a lot of _features_ listed in the docs (though support is pretty sketchy for a lot of them). I've added any that look like they could be relevant from a security point of view.